xml/security_kerberos.xml

```diff
@@ -1914,6 +1914,140 @@ Valid starting     Expires            Service principal
   </procedure>
   </sect1> -->
 
+  <sect1 xml:id="sec-security-kerberos-troubleshooting">
+    <title>Troubleshooting &krb;</title>
+      <para>
+     Troubleshooting &krb; issues can be complex because of its role in secure authentication within a network.
```

Comment:
Troubleshooting kerberos is complex, because kerberos is a dumpster fire of a service.

I don't think we shuold say this, and this line adds no value to the documentation.